### PR TITLE
Feature page for QEMU Guest Agent integration

### PR DESCRIPTION
**Pull request by nyoxi**
Signed-off-by: Tomáš Golembiovský <tgolembi@redhat.com>

### CODE REVIEW

File: source/develop/release-management/features/virt/qemu-ga.html.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: QEMU Guest Agent
```
Comment (mykaul) [inline source/develop/release-management/features/virt/qemu-ga.html.md]: integration

File: source/develop/release-management/features/virt/qemu-ga.html.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+category: feature
+authors: tgolembi
+feature_name: QEMU Guest Agent
+feature_modules: vdsm
+feature_status: In Development
+---
+
+# QEMU Guest Agent
```
Comment (mykaul) [inline source/develop/release-management/features/virt/qemu-ga.html.md]: integration

File: source/develop/release-management/features/virt/qemu-ga.html.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Owner
+
+*   Name: Tomáš Golembiovský
+*   Email: tgolembi@redhat.com
+
+## Summary
+
+This feature is about introducing the most essential features of oVirt Guest
+Agent into QEMU Guest Agent (QGA) and using it VDSM.
```
Comment (mykaul) [inline source/develop/release-management/features/virt/qemu-ga.html.md]: missing 'in' (so 'using it in VDSM' at the end)

File: source/develop/release-management/features/virt/qemu-ga.html.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* power cycle commands -- shutdown, restart (oVirt 4.2)
+* host name (QGA 2.10, oVirt 4.2.3)
+* timezone information (QGA 2.10, oVirt 4.2.3)
+* list of active users (QGA 2.10, oVirt 4.2.3)
+* guest Operating System and architecture (QGA 2.10, oVirt 4.2.3)
+* list of network interfaces and IP addresses (oVirt 4.2.3)
+
+## Details and Differences
+
+There are no changes in the UI in the form how the information is presented.
```
Comment (mykaul) [inline source/develop/release-management/features/virt/qemu-ga.html.md]: Remove 'the form'

**Comment by nyoxi** [comment]
Updated based on comments from @mykaul

**Pull request merged**